scripts/BuildPackagesFromSigningDirectory.ps1

```diff
@@ -0,0 +1,57 @@
+<#
+.SYNOPSIS
+    Package the SARIF SDK from a custom signing directory.
```

Comment:
>custom [](start = 33, length = 6)

Why "custom"? How about "Create the SARIF SDK packages from the binaries in the signing directory."